prepare-iso.sh

```diff
@@ -26,11 +26,23 @@ function createISO()
 
     # echo Debug: installerAppName = ${installerAppName} , isoName = ${isoName}
 
+    echo
+    echo Create ${isoName} blank ISO image with a Single Partition - Apple Partition Map
+    echo --------------------------------------------------------------------------
+    # Just in case - delete any previous sparseimage
+    [ -e /tmp/${isoName}.sparseimage ] && rm -f /tmp/${isoName}.sparseimage
+    # increased size to 16G - 8G is too small for Catalina
+    echo $ hdiutil create -o /tmp/${isoName} -size 16g -layout SPUD -fs HFS+J -type SPARSE
+    hdiutil create -o /tmp/${isoName} -size 16g -layout SPUD -fs HFS+J -type SPARSE
+
```

Comment:
Note, this block was just moved up since Big Sur will mount this as the installer image, `install_app`.